docs/content/guides/dagster/graph_job_op.mdx

```diff
@@ -500,29 +497,28 @@ def do_it_all_repo():
     return [do_it_all, do_it_all_schedule]
 ```
 
-With the graph APIs, you can first define a partitioning for the job. These partitions exist independently of any schedule, and can be used for backfills. A schedule can then be derived from that partitioning, while providing execution time information. The resulting schedule can be independently passed to a repository, and the underlying job will be inferred.
+With the job APIs, you can first define a graph, and then a job with partitioning. These partitions exist independently of any schedule, and can be used for backfills. A schedule can then be derived from that partitioning, while providing execution time information. The resulting schedule can be independently passed to a repository, and the underlying job will be inferred.
 
 ```python file=/guides/dagster/graph_job_op/graph_with_partition_schedule.py
-from dagster import daily_partitioned_config, graph, op, repository, schedule_from_partitions
+from dagster import daily_partitioned_config, op, repository, schedule_from_partitions, job
 
 
 @op(config_schema={"date": str})
 def do_something_with_config(context):
     return context.op_config["date"]
 
 
-@graph
-def do_it_all():
-    do_something_with_config()
-
-
 @daily_partitioned_config(start_date="2020-01-01")
 def do_it_all_config(start, _end):
     return {"solids": {"do_something_with_config": {"config": {"date": str(start)}}}}
 
 
-do_it_all_job = do_it_all.to_job(config=do_it_all_config)
-do_it_all_schedule = schedule_from_partitions(do_it_all_job)
+@job(config=do_it_all_config)
+def do_it_all():
+    do_something_with_config()
+
+
+do_it_all_schedule = schedule_from_partitions(do_it_all)
```

Comment:
re: config weirdness
i feel, in this case, we could recommend using `@graph` over `@job`

Reply:
FWIW I think this looks fine.  If the config includes resources, then, even with graph, we'd have a situation where the config precedes the place where the resources are included.